src/blog-utils/index.js

```diff
@@ -52,6 +53,13 @@ export const buildFooter = (): string => {
   return render({ app, lang, component: <Footer noLangPicker /> });
 };
 
+export const buildHeader = (): string => {
+  const app = 'firefox';
+  const lang = 'en-US';
```

Comment:
do these need to be specified - the blog won't be app specific, will it?  (and will it be localised?)

Reply:
Those two values are the "default" values but they are not strictly defined in our code. `en-US` is the default locale/fallback, same for `firefox`. We need these values because the header for Android is different than the one for Desktop (`firefox`).

Reply:
my thinking was: if they're not actually used in building the header then they should be optional parameters

Reply:
Yes, I was also thinking about moving these default values in `render`. It's done like this in `buildFooter()` too, so I'll update both functions in a follow-up